### PR TITLE
Make the "lxc" module compile

### PR DESCRIPTION
**Pull request by dmajda**

### CODE REVIEW

File: data/lxc.yml [hunk 1]
```diff
@@ -1,8 +1,18 @@
-# TODO: Process.
+deps:
+  - yast2
+  - security
+  - pam
```
Comment (lslezak) [inline data/lxc.yml:4]: Is pam really needed?

Comment (dmajda) [inline data/lxc.yml:4]: It turns out this is a transitive dependency (a dependency of `security`). We don't handle this yet in `yk`, which we probably should. Let's keep this PR open for now and I'll implement a fix.

Comment (dmajda) [inline data/lxc.yml:4]: Transitive dependencies were implemented in #100 and the `pam` module dependency was removed in #101.

**Comment by dmajda** [comment]
Replaced by #101.

**Pull request closed without merging**